docs/api-ref/rest-api/index.md

```diff
@@ -16,6 +19,111 @@ Prefect Cloud and a locally hosted Prefect server each provide a REST API.
 - Interactive REST API documentation for a locally hosted open-source Prefect server is available at <a href="http://localhost:4200/docs" target="_blank">http://localhost:4200/docs</a> or the `/docs` endpoint of the [`PREFECT_API_URL`](/concepts/settings/#prefect_api_url) you have configured to access the server. You must have the server running with `prefect server start` to access the interactive documentation.
 - The REST API documentation for a locally hosted open-source Prefect server is also available in the [Prefect REST API Reference](/api-ref/rest-api-reference/).
 
+## Interacting with the REST API
+
+You have many options to interact with the Prefect REST API:
+
+- Create an instance of [`PrefectClient`](/api-ref/prefect/client/orchestration/#prefect.client.orchestration.PrefectClient) 
+- Use your favorite Python HTTP library such as [Requests](https://requests.readthedocs.io/en/latest/) or [HTTPX](https://www.python-httpx.org/)
+- Use an HTTP library in your language of choice
+- Use [curl](https://curl.se/) from the command line 
+
+### PrefectClient with Prefect server
```

Comment:
```suggestion
### `PrefectClient` with Prefect server
```

Reply:
Not sure of the docs style guide on headings but in the preview it looks strange to me, specifically "curl with Prefect" without the code dashes on `curl`

Reply:
curl's website rolls similarly, so I think it's ok as is. :)